Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM alpine
+COPY build/ /build
+CMD ["sh", "-c", "cp -r /build/* /opt/openchs/static/ && sleep infinity"]
```

Comment:
Will `/opt/openchs/static` exist in `alpine` . Is there an `mkdir` required in the `CMD` before the `cp` ?

Reply:
As you can see here, https://github.com/Bahmni-HWC/bahmni-india-package/commit/ee071e1ef13d7bfd50d1fe131dad0bad8b7baf0d, the volume mount will create the directory

Reply:
@mohan-13 : This image needs to exist independently of volume mounts. IMO outside of that `docker-compose.yml`, running this image will fail without the `mkdir`